cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/kuberay-operator/clusterrolebinding.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kuberay-operator
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kuberay-operator
+subjects:
+- kind: ServiceAccount
+  name: kuberay-operator
+  namespace: ray-bu
```

```suggestion
  namespace: ray-bu

```

Should be a newline here at end of file to pass our linting check. Alternatively you could run `pre-commit run -a` provided you have it installed to test all linting.